test/xmppjs/test_XJSAccount.ts

```diff
@@ -67,7 +67,8 @@ describe("XJSAccount", () => {
                 await acct.joinChat({
                     room: "den",
                     server: "remote.server",
-                }, instance as any, 50, true);
+                // Explicit any - we want to deliberately send wrong params
```

Comment:
Fair enough, but is this really necessary in the presence of typechecking?

Reply:
I think we have to assume it's possible it might still be invalid, as some of this data comes from the DB.